cluster/CHAPTER.rst

```diff
@@ -244,11 +241,11 @@ Using named tuples to describe each message type keeps the code clean and helps
 The named tuple constructor will raise an exception if it is not given exactly the right attributes, making typos obvious.
 The tuples format themselves nicely in log messages, and as an added bonus don't use as much memory as a dictionary.
 
-Creating a message reads naturally::
+Creating a message reads naturally:
```

Comment:
Cool, thanks -- I was stripping the RST without really knowing what I was doing. >_<